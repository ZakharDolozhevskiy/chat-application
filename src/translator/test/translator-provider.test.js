import React from 'react';
import { shallow } from 'enzyme';
import TranslatorProvider from '../translator-provider';

it('should render translator provider', () => {
  const component = shallow(
    <TranslatorProvider language="ENG">
      <span>Consumers</span>
    </TranslatorProvider>);

  expect(component).toMatchSnapshot();
});
