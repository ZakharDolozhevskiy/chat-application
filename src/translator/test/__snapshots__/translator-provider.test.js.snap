// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render translator provider 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TranslatorProvider
    language="ENG"
>
    <span>
        Consumers
    </span>
</TranslatorProvider>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <span>
        Consumers
</span>,
      "value": [Function],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Consumers",
      },
      "ref": null,
      "rendered": "Consumers",
      "type": "span",
    },
    "type": Object {
      "$$typeof": Symbol(react.provider),
      "_context": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": [Circular],
        "_calculateChangedBits": null,
        "_changedBits": 0,
        "_changedBits2": 0,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": "translator",
        "_currentValue2": "translator",
        "_defaultValue": "translator",
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <span>
          Consumers
</span>,
        "value": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Consumers",
        },
        "ref": null,
        "rendered": "Consumers",
        "type": "span",
      },
      "type": Object {
        "$$typeof": Symbol(react.provider),
        "_context": Object {
          "$$typeof": Symbol(react.context),
          "Consumer": [Circular],
          "Provider": [Circular],
          "_calculateChangedBits": null,
          "_changedBits": 0,
          "_changedBits2": 0,
          "_currentRenderer": null,
          "_currentRenderer2": null,
          "_currentValue": "translator",
          "_currentValue2": "translator",
          "_defaultValue": "translator",
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
